database/clicker.go

```diff
@@ -1,5 +1,123 @@
 package database
 
-func WriteDB() {
+import (
+	"bypass/getter"
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"os"
+	"path"
+	"time"
 
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+	"github.com/joho/godotenv"
+	"github.com/pkg/errors"
+)
+
+type Conf struct {
+	Database,
+	Username,
+	Password,
+	Host,
+	Port string
+}
+
+type Row struct {
+	CreatedAt      time.Time `ch:"created_at"`
+	ProductID      string    `ch:"product_id"`
+	ID             int64     `ch:"id"`
+	SearchPhraseID int32     `ch:"search_phrase_id"`
+	SearchPosition int32     `ch:"search_position"`
+}
+
+func GetEnv() (Conf, error) {
+	var config Conf
+	keys := []string{"USERNAME", "PASSWORD", "HOST", "PORT", "DATABASE"}
+	if err := godotenv.Load("./files/conf.env"); err != nil {
+		return config, errors.Wrap(err, "No .env file found")
```

Comment:
```suggestion
		return config, errors.Wrap(err, ".env file not found")
```

1. No не нужен
2. Ошибки в го начинаются на с заглавной буквы